tutorials/BurstDetection.ipynb

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
The name of this file should follow the spacing / capitalization conventions set by the other notebooks